### DISCUSSION TITLE
as a beginner, could someone help me understand this line?

### DISCUSSION THREAD

**Discussion by caozhuozi** [tailwindlabs/tailwindcss#14975] [Help]
I’m learning a project that creates a Nike-like landing page using Tailwind CSS. However, I don’t quite understand this line in [App.jsx](https://github.com/adrianhajdin/nike_landing_page/blob/main/src/App.jsx#L17). It mentions that when the screen size is `xl`, it uses `padding-l`, which is a custom style defined in [index.css](https://github.com/adrianhajdin/nike_landing_page/blob/main/src/index.css#L39).

But in that definition, it already specifies that it’s for the `sm` breakpoint:

```css
.padding-l {
    @apply sm:pl-16 pl-8;
}
```

Since `xl:sm` is not valid Tailwind syntax, I’m curious about what happens when these styles take effect together.

**Comment by mutasim77** [comment]
The base padding `pl-8` applies to all screen sizes by default, but once the screen reaches the `sm` breakpoint (640px and above), it switches to use the larger padding `pl-16` instead. 
Also when you use `@apply` in your index.css, you can create your own custom class names, like `.padding-l` that bundle multiple Tailwind utilities together.
